libs/ts-rest/core/src/lib/unknown-status-error.ts

```diff
@@ -0,0 +1,15 @@
+export class UnknownStatusError extends Error {
```

Comment:
Going to need to export this in index.ts so it's publicly accessible.

Sorry for the staggered reviews! 😅

Reply:
No worries. Loving how detailed your reviews are!